app/Database/Database.hs

```diff
@@ -35,12 +36,29 @@ setupDatabase quiet = do
     let ind = (T.length dbPath -) . fromMaybe 0 . T.findIndex (=='/') . T.reverse $ dbPath
         db = T.unpack $ T.take ind dbPath
     createDirectoryIfMissing True db
+
+    -- 1. Ensure SQL schema matches ORM (in Tables.hs)
     runDb (
         if quiet
             then void $ runMigrationQuiet migrateAll
             else runMigration migrateAll
         )
 
+    -- 2. Initialize the SchemaVersion table.
+    void $ runDb getDatabaseVersion
+
+-- | Gets the current version of the database.
+-- If no version is defined, initialize the
+-- version to 1 and return that.
+getDatabaseVersion :: MonadIO m => SqlPersistT m Int
+getDatabaseVersion = do
+    result <- selectFirst [] []
+    case result of
+        Just entity -> pure $ schemaVersionVersion $ entityVal entity
+        Nothing -> do
+            insert_ $ SchemaVersion 1
```

Comment:
let's define a variable for `1` so that there's no ambiguity that the `1` plays the same role in both this line and the next.